beam-migrate-cli/beam-migrate-cli.cabal

```diff
@@ -47,7 +47,7 @@ executable beam-migrate
                        network              >=2.6      && <2.9,
                        yaml                 >=0.8      && <0.12,
                        uuid                 >=1.3      && <1.4,
-                       hint                 >=0.6      && <0.10,
+                       hint                    >=0.6      && <0.10,
```

Comment:
Did you mean to commit this?

Reply:
no

Reply:
BTW, I use haskell-mode's standard import formatter. Not sure how to reproduce it. I'm open to adopting a styling tool, but don't know much about them.